content/en/integrations/vmware_tanzu_application_service.md

```diff
@@ -0,0 +1,163 @@
+---
+integration_title: VMware Tanzu Application Service
+name: vmware_tanzu_application_service
+kind: integration
+aliases:
+    - /integrations/cloud_foundry/
+    - /integrations/pivotal_platform/
+newhlevel: true
+updated_for_agent: 6.0
+description: 'Track the health of your VMware Tanzu Application Service (formerly Pivotal Cloud Foundry) VMs and the jobs they run.'
+is_public: true
+public_title: Datadog-VMware Tanzu Application Service (Pivotal Cloud Foundry) Integration
+short_description: 'Track the health of VMware Tanzu Application Service VMs and the jobs they run.'
+categories:
+    - provisioning
+    - configuration & deployment
+    - log collection
+dependencies:
+    ['https://github.com/DataDog/documentation/blob/master/content/en/integrations/vmware_tanzu_application_service.md']
+doc_link: /integrations/vmware_tanzu_application_service/
+integration_id: "pivotal-platform"
+---
+
+## Overview
+
+Any VMware Tanzu Application Service (formerly known as Pivotal Cloud Foundry, read the [VMware announcement][3] for more information) deployment can send metrics and events to Datadog. You can track the health and availability of all nodes in the deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more. For the best experience, use this page to automatically set up monitoring through Tanzu Ops Manager for your application on VMware Tanzu Application Service and your VMware Tanzu Application Service cluster. For manual setup steps, read the [VMware Tanzu Application Service Manual Setup Guide][6].
+
+There are three main components for the VMware Tanzu Application Service integration with Datadog. First, the buildpack is used to collect custom metrics from your applications. Second, the BOSH Release collects metrics from the platform. Third, the Loggregator Firehose Nozzle collects all other metrics from your infrastructure. Read the [Datadog VMware Tanzu Application Service architecture][32] guide for more information.
```

Comment:
```suggestion
There are three main components for the VMware Tanzu Application Service integration with Datadog. First, the buildpack is used to collect custom metrics from your applications. Second, the BOSH Release collects metrics from the platform. Third, the Loggregator Firehose Nozzle collects all other metrics from your infrastructure. For more information, see the [Datadog VMware Tanzu Application Service architecture guide][32].
```

Reply:
As before, would you be open to modifying your suggestion to incorporate `read`?